test/org/sagebionetworks/bridge/services/SendMailViaAmazonServiceParticipantRosterTest.java

```diff
@@ -69,12 +68,12 @@ public void sendParticipantRoster() {
         verify(emailClient).sendRawEmail(argument.capture());
         
         SendRawEmailRequest req = argument.getValue();
-        assertEquals("Correct sender", "test-sender@sagebase.org", req.getSource());
+        assertEquals("Correct sender", "bridge-testing+emailadmin@sagebase.org", req.getSource());
 
         // validate to
         List<String> toList = req.getDestinations();
         assertEquals("Correct number of recipients", 1, toList.size());
-        assertEquals("Correct recipient", "consent-notification@test.com", toList.get(0));
+        assertEquals("Correct recipient", "consent@acme.com", toList.get(0));
```

Comment:
Changed the default study to use:

bridge-testing+support@sagebase.org
bridge-testing+technical@sagebase.org
bridge-testing+consent@sagebase.org